x-pack/filebeat/module/aws/cloudtrail/test/add-user-to-group-json.log-expected.json

```diff
@@ -21,9 +22,6 @@
         "fileset.name": "cloudtrail",
         "input.type": "log",
         "log.offset": 0,
-        "related.user": [
```

Comment:
`related.user` is lost after this change, but can't see why it happens. Can it be fixed?